src/renderer/components/editorWithTabs/index.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <div
       class="editor-with-tabs"
+      :class="[theme]"
```

Comment:
I think this is no needed.

Reply:
Yes, it's needed. Otherwise the background is white.